Document-Processing/PDF/PDF-Viewer/maui/Print-a-Document.md

```diff
@@ -36,6 +36,24 @@ Below is an example of the print preview dialog on the Android platform. The app
 
 ![Printing PDF Files in .NET MAUI PDF Viewer](Images/Print/print.png)
 
+## How to adjust the print quality on the Windows platform?
```

Comment:
@immankumarsync. Explanation  for each enums are missing. So that user can choose their enum based on their requirement.